tests/python/relay/aot/corstone300.mk

```diff
@@ -48,6 +48,11 @@ CC = arm-none-eabi-gcc
 AR = arm-none-eabi-ar
 RANLIB = arm-none-eabi-ranlib
 CC_OPTS = CC=$(CC) AR=$(AR) RANLIB=$(RANLIB)
+ifeq ($(shell [ -d ${CMSIS_PATH}/CMSIS-NN ]; echo $$?), 0)
+	CMSIS_NN_PATH = ${CMSIS_PATH}/CMSIS-NN
+else
+	CMSIS_NN_PATH = ${CMSIS_PATH}/CMSIS/NN
+endif
```

Comment:
doesn't simply looking for the directory work `if [ -d " ${CMSIS_PATH}/CMSIS-NN" ]; ` for simplicity - just curious?

Reply:
If you want I can check and improve that, but this code will be reverted when the docker image will be updated.